README.adoc

```diff
@@ -1128,3 +1128,89 @@ https://issues.jenkins-ci.org[Jenkins issue tracker].
 
 Refer to link:CONTRIBUTING.adoc#contributing-to-the-git-plugin[contributing to the plugin] for contribution guidelines.
 Refer to link:Priorities.adoc#git-plugin-development-priorities[plugin development priorities] for the prioritized list of development topics.
+
+[#Remove Git Plugin BuildsByBranch BuildData Script]
+== Remove Git Plugin BuildsByBranch BuildData Script
+
+This script is used to remove the static list of BuildsByBranch that is
+uselessly stored for each build by the Git Plugin.
+
+* This is a workaround for the problem described here:
+https://issues.jenkins-ci.org/browse/JENKINS-19022
+* Updated to handle Matrix Project types.
+* Updated to better support SCM Polling
+* Updated to handle Projects inside Folders.
+* Updated to handle Pipeline job types (just call getJobNames() to find
+everything)
+
+[source,syntaxhighlighter-pre]
```

Comment:
Since the script is groovy syntax, the layout is a little more attractive if we declare the script is groovy
```suggestion
[source,groovy]
```